services/searchUser.js

```diff
@@ -0,0 +1,168 @@
+export const fetchUsers = () => {
```

Comment:
Should be renamed to `fetchConsultants` instead of `fetchUsers` for better code readability. Also should be moved to the `service/user.js` - we put any service functions related to fetching/modifying user data there

Reply:
@quanphan2906 sounds good I will make this change

Reply:
I noticed you renamed the function but haven't moved it to `service/user.js`. Did you miss this or is there anything that concerns you?

Reply:
My bad I just missed it. lol for some reason the last part just skipped my mind when i was doing it